server/src/test/java/org/apache/uniffle/server/buffer/ShuffleBufferManagerTest.java

```diff
@@ -587,6 +587,7 @@ public void shuffleFlushThreshold() throws Exception {
     shuffleBufferManager.cacheShuffleData(appId, smallShuffleId, false, createData(0, 31));
     assertEquals(96 + 63, shuffleBufferManager.getUsedMemory());
     shuffleFlushManager.flush();
+    Thread.sleep(100);
```

Comment:
Why do we need to sleep here?

Reply:
server/src/test/java/org/apache/uniffle/server/TestShuffleFlushManager.java#createFlushEventExecutor  method make here sync, so we do not sleep here before.